src/mach/exports.rs

```diff
@@ -294,6 +294,30 @@ impl<'a> ExportTrie<'a> {
             location,
         }
     }
+
+    /// Create a new, lazy, zero-copy export trie from the `LinkeditDataCommand` `command`
+    pub fn new_from_linkedit_data_command(
```

Comment:
Presumably, we don't want a variation of the `new` function, but was simple to implement. What do you think?

Reply:
The name is kind of long, but fine; however, can you dispatch both of these implemenations to a new private constructor, say `new_impl` which takes the bytes + the offset + size, since that is how they only differ, e.g.:

```rust
fn new_impl(bytes: &'a [u8], offset: usize, size: usize) -> Self {
 // etc.
}

pub fn new(bytes: &'a [u8], command: &load_command::DyldInfoCommand) -> Self {
  Self::new_impl(bytes, command.export_off as usize, ommand.export_size as usize)
}

// one more using the LinkeditDataComand
```

Reply:
@m4b Updated.